docs/data/sources/google-tag-manager-client-legacy.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Amplitude Analytics Legacy
+description: Official Google Tag Manager template for Amplitude Analytics for data collection.
+---
+
+Client-side Google Tag Manager Template for Amplitude Analytics. The tag utilizes the [Amplitude JavaScript SDK](../../data/sdks/javascript/index/) for data collection.
```

Comment:
```suggestion
Client-side Google Tag Manager Template for Amplitude Analytics. The tag uses the [Amplitude JavaScript SDK](../../data/sdks/javascript/index/) for data collection.
```

Protip: choose "use" over "utilize" in almost every case.

Reply:
Changed "utilize" to "use".